asterisk_click2dial/models/asterisk_server.py

```diff
@@ -146,29 +146,33 @@ def _get_connect_info(self, url_path):
         user = self.env.user
         ast_server = user.get_asterisk_server_from_user()
         auth = (ast_server.login, ast_server.password)
-        url = "http://%s:%s%s" % (ast_server.ip_address, ast_server.port, url_path)
+        url = f"http://{ast_server.ip_address}:{ast_server.port}{url_path}"
         return ast_server, auth, url
 
     def test_ari_connection(self):
         self.ensure_one()
         auth = (self.login, self.password)
-        url = "http://%s:%s/ari/asterisk/info" % (self.ip_address, self.port)
+        url = f"http://{self.ip_address}:{self.port}/ari/asterisk/info"
         try:
             res = requests.get(url, auth=auth, timeout=TIMEOUT)
         except Exception as e:
             raise UserError from None(
```

Comment:
I have models/asterisk_server.py", line 159, in test_ari_connection
TypeError: 'NoneType' object is not callable

Reply:
Thanks for your review, comments attended!